src/app/modules/forms/ix-forms/components/ix-button-group/ix-button-group.component.ts

```diff
@@ -41,6 +41,8 @@ export class IxButtonGroupComponent implements ControlValueAccessor {
   @HostBinding('class.inlineFields')
   @Input() inlineFields = false;
 
+  formControlName = input<string>();
```

Comment:
Did you want to use `this.controlDirective.name` or `label` input?

Reply:
`this.controlDirective.name` is null/undefined even after `ngAfterViewInit`